app-emulation/uade/files/uade-2.13-configure.patch

```diff
@@ -0,0 +1,40 @@
+--- uade-2.13/configure	2009-10-29 22:01:12.000000000 +0100
++++ uade-2.13-r1/configure	2020-06-29 22:21:16.393205188 +0200
+@@ -337,13 +337,17 @@
+     fi
+ fi
+ 
++if test -z "$libdir" ; then
++    libdir="$prefix/lib"
++fi
```

Comment:
The problem with putting this here is that `$prefix` may not have been defined yet. You'll need to put it twice into the conditional block below.

Reply:
Ahh, yes, you're right. Haven't thought about that.
I'm going to update this tomorrow (as well as the other issues) - it's already late here. Thanks for reviewing.